cardano-db/src/Cardano/Db.hs

```diff
@@ -7,14 +7,13 @@ module Cardano.Db (
   TxIn (..),
   TxOut (..),
   gitRev,
-  migrateTxOut,
 ) where
 
 import Cardano.Db.Delete as X
 import Cardano.Db.Error as X
 import Cardano.Db.Insert as X
 import Cardano.Db.Migration as X
-import Cardano.Db.Migration.Extra.CosnumedTxOut.Queries (migrateTxOut)
```

Comment:
I would prefer to keep the scope of the Extra.Migration queries as limited as possible, since it defines queries on a different schema, that can cause issues if executed against the default schema.

Reply:
cool, I'll just export the function I need for the tests 👍